renderdoc/driver/vulkan/vk_outputwindow.cpp

```diff
@@ -401,24 +394,33 @@ void VulkanReplay::OutputWindow::Create(WrappedVulkan *driver, VkDevice device,
 
     GetResourceManager()->WrapResource(Unwrap(device), swap);
 
+    uint32_t numImgs;
```

Comment:
For safety here can you initialise numImgs to 0? Just in case the driver doesn't write to it for some reason. This will probably crash later on but at least we won't immediately die resizing the array to some invalid size.